elb/status/status.go

```diff
@@ -42,14 +43,19 @@ type status struct {
 	awsElb           elb.ELB
 	labelValue       string
 	kubernetesClient k8s.Client
-	initialised      sync.Mutex
+	clusterFrontEnds map[string]elb.LoadBalancerDetails
```

Comment:
would it make sense to have a consistent name for the LoadBalancer for both elb and merlin?
for some reason merlin has a simplified `lb`